CONTRIBUTING.md

```diff
@@ -63,3 +63,9 @@ It's last chance to check your code before creating PR :)
 
 When PR is created, don't forget to send a link to slack:
 ![Screenshot 3](https://i.imgur.com/eA1zFQt.png)
+
+# How to merge PR
+Once your PR is approved and all CI checks have passed, you can merge your PR.  
```

Comment:
Please add:
1. Make sure your PR title and description match what you actually did.
2. Please name the commit title the same as the PR.